config/configuration.go

```diff
@@ -50,6 +50,9 @@ const (
 	//PipelineSystemNamespace pipeline infra namespace key
 	PipelineSystemNamespace = "infra.namespace"
 
+	//PipelineHeadNodePoolName name of our Head node pool for Pipeline Infra deployments
+	PipelineHeadNodePoolName = "infra.headNodePoolName"
```

Comment:
Please set a deafult value for this

Reply:
I've left this intentionally empty because if you don't set this property Pipeline will not attempt to deploy Infra components to a reserved node pool.

Reply:
okay, thx